src/private/files.ts

```diff
@@ -226,6 +226,43 @@ export namespace files {
     listUrl?: string;
   }
 
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Download Files interface
+   */
+  export interface IFileItem {
+    /**
+     * ID of the file metadata
+     */
+    objectId?: string;
+    /**
+     * Path of the file
+     */
+    path?: string;
+    /**
+     * Size of the file
+     */
+    size?: number;
```

Comment:
>size [](http://example.com/codeflow?start=4&length=4)

This should indicate what the unit is (bytes for example)

Reply:
Updated the doc comment.

Reply:
I'd rather have the property be called `sizeInBytes` instead of this being in a comment.

Reply:
Since the property is named as **size** in Teams Web Client, hence retained the same name here, so that the conversion/switch can happen seamlessly between SDK and Teams Web Client code.
@ydogandjiev Please correct me if it can work otherwise, as suggested by Marcin.

Reply:
As discussed offline, this is not good enough of a reason for me. We need to focus on designing the best possible developer experience for the developers using the SDK. Not guide the design based on implementation details of Teams (or any other host) trying to host these.

Reply:
Suggestion incorporated.